### PR TITLE
Adjust task names, tags and variables to the Adfinis standard

### PR DESCRIPTION
**Pull request by karras**
##### SUMMARY

Adjust task names, tags and variables to the Adfinis standard 

##### ISSUE TYPE

 - Feature Pull Request

##### ANSIBLE VERSION

```
n/a
```

##### ADDITIONAL INFORMATION

```
n/a
```

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -6,16 +6,16 @@
...
+    - 'role::openshift_backup'
+    - 'role::openshift_backup:install'
+    - 'role::openshift_backup:config'
 
 - import_tasks: installation.yml
   tags:
-    - 'role::$role'
-    - 'role::$role:install'
+    - 'role::openshift_backup'
+    - 'role::openshift_backup'
```
Comment (tongpu) [inline tasks/main.yml]: 2nd line should probably be `role::openshift_backup:install`?

**Review by tongpu — CHANGES_REQUESTED** [review]
LGTM besides the minor tag issue

**Comment by karras** [comment]
Fixed

**Review by tongpu — APPROVED** [review]
LGTM

**Pull request merged**